tox.ini

```diff
@@ -8,23 +8,20 @@
 skipsdist = true
 envlist = py{26,27,33,34,35},py{26,27}-oldest,cover,lint
 
-# nosetest -v => more verbose output, allows to detect busy waiting
-# loops, especially on Travis
```

Comment:
so why is `nosetest` actually removed? If it is removed here, why do we keep `'nose'` in `testing_extras` throughout?